sc/src/main/webapp/WEB-INF/jsp/tiles/views/gene-search-results.jsp

```diff
@@ -5,17 +5,34 @@
 
 <%@ page contentType="text/html;charset=UTF-8" %>
 
+<link rel="stylesheet" href="http://localhost:8080/gxa/sc/resources/css/popup-feedback.css" type="text/css" media="all">
```

Comment:
Same comments as for the `experiment-page.jsp`. You do not need to add the CSS to every JSP, as they all inherit from `base.jsp`.